quetz/main.py

```diff
@@ -1559,14 +1556,20 @@ async def serve_path(
     path,
     channel: db_models.Channel = Depends(get_channel_allow_proxy),
     accept_encoding: Optional[str] = Header(None),
-    cache: LocalCache = Depends(LocalCache),
     session=Depends(get_remote_session),
     dao: Dao = Depends(get_dao),
 ):
 
     if channel.mirror_channel_url and channel.mirror_mode == "proxy":
         repository = RemoteRepository(channel.mirror_channel_url, session)
-        return get_from_cache_or_download(repository, cache, path)
+        if not pkgstore.file_exists(channel.name, path):
+            download_remote_file(repository, pkgstore, channel.name, path)
+        elif path.endswith(".json"):
+            # repodata.json and current_repodata.json are cached locally
+            # for channel.ttl seconds
+            _, fmtime, _ = pkgstore.get_filemetadata(channel.name, path)
+            if time.time() - fmtime >= channel.ttl:
```

Comment:
I think using the channel ttl makes sense here but I noticed I couldn't change the default value when defining the proxy channel.
The default is 10 hours which is probably too long for this use case. Is that on purpose?